problem2.py

```diff
@@ -1 +1,39 @@
+"""
+----------------------------------------------------------
+Name:    problem2.py
+Purpose: Write a program that allows Tony to enter the distance driven for each day until the total distance driven surpasses 100km
 
+Author:  Hosey.K
+
+Created: date in 03/03/2021
+----------------------------------------------------------
+"""
+
+# Header of tracker
+print("~~~~~~~~~ Welcome to the DoorDash Distance Tracker ~~~~~~~~~")
+print("")
+
+# Travel log title
+print("~~~~~~ Travel Log ~~~~~~")
+
+# Input distance travelled
+number_days = 0
+distance_per_day = 0 
+total_distance = 0
+
+while distance_per_day < 100:
```

Comment:
incorrect logic here. should be   `total_distance <= 100` instead of `distance_per_day < 100`